modules/core/src/main/scala/io/chrisdavenport/cats/time/instances/monthday.scala

```diff
@@ -0,0 +1,19 @@
+package io.chrisdavenport.cats.time.instances
+
+import java.time.MonthDay
+import java.time.format.DateTimeFormatter
+
+import cats.{ Hash, Order, Show }
+
+trait monthday {
+  implicit final val monthdayInstances =
+    new Show[MonthDay] with Order[MonthDay] with Hash[MonthDay] {
+      def compare(x: MonthDay, y: MonthDay): Int = x.compareTo(y)
+
+      def hash(x: MonthDay): Int = x.hashCode
+
+      def show(x: MonthDay): String = x.format(DateTimeFormatter.ofPattern("M-d"))
```

Comment:
@LukaJCB does this match the formatting you were expecting?

Reply:
I don't really have a preference `"M-d"` is fine for me :+1: :)